PowerPointLabs/PowerPointLabs/Utils/Graphics.cs

```diff
@@ -53,9 +53,19 @@ public static Shape CorruptionCorrection(Shape shape, PowerPointSlide ownerSlide
 
         public static void ExportShape(Shape shape, string exportPath)
         {
-            var slideWidth = (int)PowerPointPresentation.Current.SlideWidth;
-            var slideHeight = (int)PowerPointPresentation.Current.SlideHeight;
-
+            int slideWidth = 0;
+            int slideHeight = 0;
+            try
+            {
+                slideWidth = (int)PowerPointPresentation.Current.SlideWidth;
+                slideHeight = (int)PowerPointPresentation.Current.SlideHeight;
+            }
+            catch (NullReferenceException)
+            {
+                // Presentation may be null during Unit Tests

```

Comment:
The `NullReferenceException` was thrown from the getter body of `PowerPointPresentation.Current` when it tries to reference some other objects, so I think catching the exception is the better design for this ExportShape function.
(My comment there about Presentation being null isn't very accurate, i should change it.)